### ISSUE TITLE
There are a bunch of non-GPL photographs in this repository!

### ISSUE THREAD

**Issue by gilbertohasnofb** [gilbertohasnofb/c172p-detailed-liveries#4]
There are a bunch of non-GPL photographs in this repository! They must be removed from the repo as well as from its history...

Apologies to all, this is totally my mistake. This repository was based on a simple folder which I used to save my xcf work and where I also kept the reference photos of all these liveries.

**Comment by gilbertohasnofb** [comment]
@onox what is the best approach to this? Should I first delete those on my local branch then do a `git push` before tackling the history of the repo?

**Comment by onox** [comment]
You could do some interactive rebasing and then (if `git log` is ok) force push. Or use [BFG Repo-Cleaner](https://rtyley.github.io/bfg-repo-cleaner/) to delete the blobs of the files.

**Comment by gilbertohasnofb** [comment]
I will try the latter, as those were introduced long time ago and probably in a mixed commit with a lot of other stuff, so rebasing might be a nightmare.

**Comment by onox** [comment]
Then you should look at BFG. I used to delete the big `Resources/` folder in the p51d repository.

**Comment by gilbertohasnofb** [comment]
Great, I will study how it works later on tonight.

**Comment by gilbertohasnofb** [comment]
@onox You know what, this repository is nearly 4Gb in total size even though the current files themselves total to only 1Gb. I am so considering to simply delete this repository permanently and then create a new one with an identical name, thus solving this issue here as well as bringing the size of this repo down once again. Do you think that's a good solution or do you find it too heavy handed? Other than my own commits there are only a single commit by @wlbragg and another single one by @wkitty42 so it's not much what would be lost in terms of history.

**Comment by wkitty42** [comment]
On 03/05/2018 02:31 PM, Gilberto Agostinho wrote:
> @onox <https://github.com/onox> You know what, this repository is nearly 4Gb in 
> total size even though the current files themselves total to only 1Gb. I am so 
> considering to simply delete this repository permanently and then create a new 
> one with an identical name, thus solving this issue here as well as bringing the 
> size of this repo down once again. Do you think that's a good solution or do you 
> find it too heavy handed? Other than my own commits there are only a single 
> commit by @wlbragg <https://github.com/wlbragg> and another single one by 
> @wkitty42 <https://github.com/wkitty42> so it's not much what would be lost in 
> terms of history.


i don't really have a problem with it...

FWIW: this is the problem with hosting binary files in a repo... there isn't a 
good diff so the whole binary file is stored each time a change is made... way 
back in the day, we (in another land) had a good binary diff and released new 
software versions via binary patch...

**Comment by onox** [comment]
@gilbertohasnofb How much is the history worth? If you think a smaller file size is much more important than being able to dig up old files, then maybe recreating the repository is an acceptable solution.

**Comment by onox** [comment]
There's also [git lfs](https://git-lfs.github.com/). I have never used it though.

**Comment by wkitty42** [comment]
On 03/05/2018 02:31 PM, Gilberto Agostinho wrote:
> @onox <https://github.com/onox> You know what, this repository is nearly 4Gb in 
> total size even though the current files themselves total to only 1Gb. I am so 
> considering to simply delete this repository permanently and then create a new 
> one with an identical name, thus solving this issue here as well as bringing the 
> size of this repo down once again. Do you think that's a good solution or do you 
> find it too heavy handed? Other than my own commits there are only a single 
> commit by @wlbragg <https://github.com/wlbragg> and another single one by 
> @wkitty42 <https://github.com/wkitty42> so it's not much what would be lost in 
> terms of history.

you could also just start a new repo... rename this one to what it is and tack 
on something 201803 to show the last year and month it was active... then just 
start a new one in your account and we can connect in to that one as desired...

TLDR; leave this one for historical purposes... just change its name and start a 
new one with this existing name or even a new one if desired...

**Comment by wkitty42** [comment]
OHWAIT! sorry to post again... i just dropped in to git hub and now, after rereading the original post, i see there's a problem with non-GPL images... i thought everything in here was your work for the c172p project and there were all your work... i don't know what to say now other than removing the non-GPL stuff would likely be easier to just go ahead and start anew... no complaints from me either way...

**Comment by gilbertohasnofb** [comment]
@onox @wkitty42 Thanks for the input, guys. I don't think the history of this repo is particularly important. It's of course nice to be able to go back in time with the xcf files, but since I work mainly with layers in GIMP then it's rare that something gets changed forever anyway. I think the best course of action is simply to delete this repo and create a new clean one. I will do that tomorrow, and I will keep the same name and address as not to break any links in our main repo or in the forum.
